src/libraries/Microsoft.PowerFx.Interpreter/CustomFunction/CustomFunctionSignatureHelper.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.PowerFx
+{
+    public class CustomFunctionSignatureHelper
```

Comment:
>CustomFunctionSignatureHelper [](http://example.com/codeflow?start=17&length=29)

A new class that will be used to provide information regarding signatures.

For now only limited to arg description, can potentially extend in future.